src/DatePicker/DatePicker.tsx

```
@@ -0,0 +1,347 @@
+    /** Placeholder text for the trigger input. */
+    placeholder?: string;
+    /** The locale to use for formatting dates (e.g., 'en-US', 'de-DE'). Defaults to browser's locale. */
+    locale?: string;
+    /** The range of years to display in the year dropdown. */
+    yearRange?: { from: number; to: number };
+}
+// #endregion
+
+// #region Icons
+const CalendarIcon = () => (
+    <svg xmlns="http://www.w3.org/2000/svg" width="16" height="16" viewBox="0 0 24 24" fill="none" stroke="currentColor" strokeWidth="2" strokeLinecap="round" strokeLinejoin="round">
```

Biome (2.1.2)

[error] Alternative text title element cannot be empty

For accessibility purposes, SVGs should have an alternative text, provided via title element. If the svg element has role="img", you should add the aria-label or aria-labelledby attribute.

(lint/a11y/noSvgWithoutTitle)